src/datasets.py

```diff
@@ -206,15 +207,15 @@ def pre_process(self):
                                 except KeyError:
                                     # print('An excluded word:', alphabetic_word)
                                     pass
-                            vectors.append(torch.from_numpy(np.array(word_vectors, np.long)))
+                            vectors.append(torch.from_numpy(np.array(word_vectors, np.long)).type(torch.LongTensor))
```

Comment:
여기에 버그 있는 게 맞나요?
```
>>> import torch
>>> import numpy as np
>>> a = np.array([1, 2, 3, 4], np.long)
>>> a
array([1, 2, 3, 4])
>>> torch.from_numpy(a)
tensor([1, 2, 3, 4])
>>> b = torch.from_numpy(a)
>>> b.type()
'torch.LongTensor'
```

`np.long`을 해놔서 inherit 될거 같은데요...

Reply:
저도 그렇게 생각했는데 아니더라고요

Reply:
제가 올려드린 snippet에선 문제 없이 LongTensor 찍히잖아요? 정확히 문제 있는 부분을 체크해 보셔야 할 것 같네요

Reply:
아마 os specific한 문제인 거 같은데, 수정하시더라도
```suggestion
vectors.append(torch.from_numpy(np.array(word_vectors)).long())
```
으로 해주세요

Reply:
올려주신 snippet 그대로 치면 놀랍게도.. 짜잔!
```
>>> import torch
>>> import numpy as np
>>> a = np.array([1, 2, 3, 4], np.long)
>>> a
array([1, 2, 3, 4])
>>> torch.from_numpy(a)
tensor([1, 2, 3, 4], dtype=torch.int32)
>>> b = torch.from_numpy(a)
>>> b.type()
'torch.IntTensor'
```

`.long()`으로 고쳐서 돌려보니까 잘 돌아갑니다. 그렇게 수정할게요.

Reply:
Done